urlchecker: Add option to strip query parameters (#182)
flathub/flatpak-external-data-checker: README.md

  version. (This follows the convention used by
  [`debian/watch`](https://wiki.debian.org/debian/watch) files.)

Some upstream vendors may add unwanted GET query parameters to

> the download URL, e.g. indentifiers for counting unique downloads.

Reviewer: ```suggestion
the download URL, such as identifiers for counting unique downloads.
```

This may result in URL change without change in the actual data.

> To remove GET query, set `strip-query` proprety to `true`.

Reviewer: ```suggestion
To remove GET query parameters, set `strip-query` property to `true`.
```
